### PR TITLE
Semantic HTML

### LINKED ISSUE #1775: Semantic HTML
### Description

I am creating issue for semantic html.

### Domain

Frontend Dev HTML/CSS/JS

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by harshapriyaa**
<hr>

## Description 📜

Added Semantic HTML documentation
Fixes #1775

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [x] Frontend Dev HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] Java#Script
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by harshapriyaa** [comment]
@code-reaper08 Please do check my contribution.

File: Web_Development/FrontEnd/Semantic_HTML/semantic_html.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+| ```<nav>```  | Used for navigation links |
+| ```<section>```  | Used for section in a document |
+| ```<summary>```  | Used for visible heading for a ```<details>``` element |
+| ```<time>```  | Used for adate/time |
+
+### References  
+    https://developer.mozilla.org/en-US/docs/Glossary/Semantics
+    https://www.w3schools.com/html/html5_semantic_elements.asp
+    photo: 
+    https://miro.medium.com/max/1000/1*zHJFnu7QF-PgUb8108aLcA.png
```
Comment (code-reaper08) [inline Web_Development/FrontEnd/Semantic_HTML/semantic_html.md:152]: This thing looks like code-block. Try to remove that.

**Review by code-reaper08 — CHANGES_REQUESTED** [review]
I've left a review. Look into that :-)

File: Web_Development/FrontEnd/Semantic_HTML/semantic_html.md (hunk 1 above)
Comment (harshapriyaa) [reply Web_Development/FrontEnd/Semantic_HTML/semantic_html.md]: I have made the changes.

**Review by code-reaper08 — COMMENTED** [review]
Looks cool!

**Pull request closed without merging**